src/App.js

```diff
@@ -1,12 +1,31 @@
 import BookList from './components/book/BookList';
 import './App.css';
+import { Link, Route, Switch } from 'react-router-dom';
+import BookDetail from './components/book/BookDetail';
 
 function App() {
   // TODO: Add routes to books & views
   return (
     <main className="container">
       <h1>Library Catalog</h1>
-      <BookList />
+      <h2>I see that you have made it to my catalog</h2>
+      <h3>Click the link below to browse the catalog</h3>
+      <Switch>
+        <Route path="/books/:id">
+          <BookDetail />
+        </Route>
+
+        <Route path="/books">
+          <Link exact to="/">
+            Home
+          </Link>
+          <BookList />
+        </Route>
+
+        <Route path="/">
```

Comment:
this is where you'd want to put that component i mentioned, and it would have the h2, h3, and this link